src/middleware/validation.js

```diff
@@ -0,0 +1,67 @@
+import Joi from 'joi';
+
+export default {
+  /**
+     * @description validates request body before registration
+     * @param {object} req
+     * @param {object} res
+     * @param {object} next
+     * @returns {object} validation oject
+     */
+
+  signupValidation(req, res, next) {
+    const userSchema = Joi.object().keys({
+      firstName: Joi.string()
+        .alphanum()
+        .min(4)
+        .max(20)
+        .required(),
+      lastName: Joi.string()
+        .alphanum()
+        .min(4)
```

Comment:
Some persons have a name with two character for example: Jack Ma

Reply:
Cool! changes incoming...